build/build.cmd

```diff
@@ -1,91 +0,0 @@
-@REM Copyright (c) Microsoft. All rights reserved.
-@REM Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-@setlocal
-@echo off
-
-set node-root=%~dp0..
-rem // resolve to fully qualified path
-for %%i in ("%node-root%") do set node-root=%%~fi
-
-rem ---------------------------------------------------------------------------
-rem -- parse script arguments
-rem ---------------------------------------------------------------------------
-
-set min-output=0
-set integration-tests=0
-
-:args-loop
-if "%1" equ "" goto args-done
-if "%1" equ "--min" goto arg-min-output
-if "%1" equ "--integration-tests" goto arg-integration-tests
-call :usage && exit /b 1
-
-:arg-min-output
-set min-output=1
-goto args-continue
-
-:arg-integration-tests
-set integration-tests=1
-goto args-continue
-
-:args-continue
-shift
-goto args-loop
-
-:args-done
-
-rem ---------------------------------------------------------------------------
-rem -- create x509 test device
-rem ---------------------------------------------------------------------------
-if "%OPENSSL_CONF%"=="" (
-  echo The OPENSSL_CONF environment variable must be defined in order to generate the x509 certificate for the test device.
-  set ERRORLEVEL=1
-  goto :eof
-)
-
-set IOTHUB_X509_DEVICE_ID=x509device-node-%RANDOM%
-call node %node-root%\build\tools\create_device_certs.js --connectionString %IOTHUB_CONNECTION_STRING% --deviceId %IOTHUB_X509_DEVICE_ID%
-set IOTHUB_X509_CERTIFICATE=%cd%\%IOTHUB_X509_DEVICE_ID%-cert.pem
-set IOTHUB_X509_KEY=%cd%\%IOTHUB_X509_DEVICE_ID%-key.pem
```

Comment:
E2E relies on this device and these variables.  Do we need to create/delete these somewhere else?

Reply:
RIght now I call them in the pipeline script. i'm going to remove those eventually though. 

---
In reply to: [200814600](https://github.com/Azure/azure-iot-sdk-node/pull/334#discussion_r200814600) [](ancestors = 200814600)